File: problemas/1100/1149.cs
using System;

namespace uri1149
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            int a = int.Parse(entrada[0]);
            int n = int.Parse(entrada[entrada.Length - 1]);
            var soma = 0;

            for (var i = 0; i < n; i++)
            {
                soma += a + i;
            }

            Console.WriteLine(soma);
        }
    }
}

File: problemas/1100/1149.dart
import 'dart:io';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = int.parse(entrada.first);
  final n = int.parse(entrada.last);
  int soma = 0;

  for (var i = 0; i < n; i++) {
    soma += a + i;
  }

  stdout.writeln(soma);
}